docs/basics/sso/migration-from-deprecated-sso.md

```diff
@@ -0,0 +1,75 @@
+| No Unique Identifier String (UIS) | In the new SSO, there is no need to specify the [Unique Identifier String](/basics/acct-team-mgmt/org-settings/#single-sign-on-settings-deprecated-flow) which was used in the legacy SSO to generate usernames for new users.                                                                                                                                                                                                                                                                                                                                                             |
+| Team Placement                    | New users are provisioned in the [default team](/basics/sso/setting-up-sso/#team-placement) in the new SSO, unlike the legacy SSO where you can specify a team for new user provisioning.                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+## Migration Tutorial
+
+### Prerequisites
+
+1. Before starting the migration process, ensure that your organization is currently using the deprecated SSO integration. To confirm this, follow the steps provided in [this guide](/basics/sso-deprecated/setting-up-single-sign-on/#how-do-i-know-if-my-organization-uses-the-deprecated-sso-integration).
+
+2. Next, on Sauce Labs click **Account**, and then click **Team Management**.
+
+<img src={useBaseUrl('img/basics/sso/setup-sso-sp-tm-menu.png')} alt="Team Management Menu" width="300" />
+
+2. On the **Organization Management** page, in the **ORGANIZATION SETTINGS** section, click **View Settings**.
+
+<img src={useBaseUrl('img/basics/sso/setup-sso-sp-org-settings-btn.png')} alt="Organization Setting Button" width="800" />
+
+3. On the **Organization Settings** page, click the **SINGLE SIGN-ON** tab.
+
+<img src={useBaseUrl('img/basics/sso/setup-sso-sp-sso-tab.png')} alt="SSO Tab" width="400" />
+
+4. If your organization is eligible for migrating to the new SSO, you will see the following banner. Please note that the migration wizard may not be enabled for your organization yet. If you don't see the banner, it means our development team is still working on enabling the migration wizard for your organization.
```

Comment:
how long should the user wait to let us know that the banner is not available? Is there a timeframe or a contact for this? 
```suggestion
4. If your organization is eligible for migrating to the new SSO, you will see the following banner. Please note that the migration wizard may not be enabled for your organization yet. If you don't see the banner, it means our development team is still working on enabling the migration wizard for your organization.
```

Reply:
The first group (almost 300 organizations) which will go through "the seamless migration" will see the banner soon, that is I hope in two weeks. The automatic migration for other organizations (less than 100) is not yet prepared. There are some extra actions that need to be taken before these organization can be qualified for "the seamless migration". The timeframe is not yet defined.